e2e/cypress/integration/ui/settings/01_user_mgmt.spec.ts

```diff
@@ -45,6 +45,7 @@ describe('user management', () => {
     // save new user
     cy.get('[data-cy=save-user]').click();
     cy.get('app-user-management chef-modal').should('not.be.visible');
+    cy.get('#main-content-wrapper').scrollTo('top');
     cy.get('chef-notification.info').should('be.visible');
```

Comment:
Before each time we check for the notification bar we need to scroll to the top because the notification bar could be hidden.